static/app/views/performance/transactionSummary/content.tsx

```diff
@@ -394,6 +428,44 @@ class SummaryContent extends React.Component<Props, State> {
   }
 }
 
+type MaxSpanQueryProps = {
+  location: Location;
+  eventView: EventView;
+  orgSlug: string;
+  referrer?: string;
+  currentFilter?: SpanOperationBreakdownFilter;
+  children: (data: number | undefined) => React.ReactNode;
+};
+
+function MaxSpanQuery(props: MaxSpanQueryProps) {
+  const {location, eventView, orgSlug, referrer, currentFilter, children} = props;
+  if (currentFilter && currentFilter !== SpanOperationBreakdownFilter.None) {
+    const maxDurationEventView = eventView
+      .withSorts([{kind: 'desc', field: 'timestamp'}])
+      .withColumns([
+        {kind: 'field', field: `spans.${currentFilter}`},
+        {kind: 'field', field: `timestamp`},
+      ]);
+    return (
+      <DiscoverQuery
+        eventView={maxDurationEventView}
+        orgSlug={orgSlug}
+        location={location}
+        referrer={referrer}
+        limit={100}
+      >
```

Comment:
I'm not convinced that this is the right query here.
1. sorting by timestamp is basically saying give me 100 events and take the largest one (more on "largest" coming up)
2. taking the max of `spans.${currentFilter}` feels incorrect since this will no longer render the bar as a percentage of the transaction duration.
3. using limit 100 here seems wasteful if we're just going to discard 99 of them

I think what we really want here is either
1. `max(transaction.duration)` (but this has problems with outliers)
2. sort by `transaction.duration`, limit 1 offset 100 and get the `transaction.duration` (I believe this is what we settled on)

Reply:
Also, I realize the second approach I described is going to have problems when there are less than 100 events 🤔

Reply:
Hey Tony, for the second approach, wouldn't that retrieve the 100th longest duration? I think what we're looking for might be the longest duration within the 100 most recent events. It might not always be the case that this is the 100th longest duration. I maybe understanding the requirements or the limit 1 offset 100 wrong so let me know if I'm off here. 
Agreed on the limit 100 being wasteful though, I landed on this because I couldn't see a way to get `DiscoverQuery` to sort by timestamp, limit 100, then sort by op duration and grab the topmost duration.